packages/@react-aria/link/src/useLink.ts

```diff
@@ -56,14 +57,15 @@ export function useLink(props: AriaLinkOptions, ref: RefObject<HTMLElement>): Li
       tabIndex: !isDisabled ? 0 : undefined
     };
   }
-
+  let {focusableProps} = useFocusable(props, ref);
```

Comment:
I'm almost surprised that FocusRing isn't the one to handle this?

Reply:
Well, there is the case where the user is making their own Link component and may not use `FocusRing` so we'd really wanna handle the `useFocusable` call in our aria hooks where possible IMO